test/test_pty.rs

```diff
@@ -0,0 +1,36 @@
+use nix::pty::openpty;
+use nix::unistd::{read, write, close};
+
+#[test]
+fn test_openpty() {
+    // TODO: figure out the right termios settings to pass in here
```

Comment:
Any advice on what flags to play with? The only ones I can think of messing with don't do much other than turn CR on/off